utils/tests/test_unauthenticated_endpoints.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+import pytest
+from django_extensions.management.commands import show_urls
+import json
+import requests
+from lxml import html
+import mvj.urls
+
+
+def paths():
+    cmd = show_urls.Command()
+    views_str = cmd.handle(
+        no_color=True,
+        language=None, decorator=None,
+        format_style='pretty-json',
+        urlconf="ROOT_URLCONF",unsorted=True)
+    views = json.loads(views_str)
+    return [path for view in views if (path := view.get('url'))]
+
+repls = (('/<pk>\.<format>/','/1234.json/'),
```

Comment:
If these variables are not reused anywhere outside the one tests, I would recommend defining them in the score of the function